doc/source/development/contributing.rst

```diff
@@ -331,7 +331,12 @@ can comment::
 
     @github-actions pre-commit
 
-on that pull request. This will trigger a workflow which will autofix formatting errors.
+on that pull request. This will trigger a workflow which will autofix formatting
+errors.
+
+To automatically fix formatting errors on each commit you make, you can
+set up pre-commit yourself. First, create a Python :ref:`environment
+<contributing.python-env>` and then set up :ref:`pre-commit <contributing.pre-commit>`.
```

Comment:
We can run `pre-commit` in a Docker container, right? If so, I think the first link should be to the https://pandas.pydata.org/pandas-docs/dev/development/contributing_environment.html page

Reply:
the command:
```
python make.py --single source/development/contributing.rst
```

Reply:
also with "quotes"

Reply:
gives this output:
```python
Traceback (most recent call last):
  File "make.py", line 377, in <module>
    sys.exit(main())
  File "make.py", line 365, in main
    builder = DocBuilder(
  File "make.py", line 56, in __init__
    single_doc = self._process_single_doc(single_doc)
  File "make.py", line 83, in _process_single_doc
    raise FileNotFoundError(f"File {single_doc} not found")
FileNotFoundError: File source/development/contributing.rst not found
```

Reply:
but `Path("source/development/contributing.rst").is_file()` inside a terminal returns `True`....

Reply:
o wow, it joined the `SOURCE_PATH` to it in make.py...

Reply:
you don't need `source/` - check the example, it has
```
python make.py --single development/contributing.rst
```